src/routes/tools/degree-days/_constants.js

```diff
@@ -0,0 +1,12 @@
+export const TOOL_SLUG = "degree-days";
```

Comment:
Like this idea of having a list of constants for the tool also

Reply:
Thanks, I think of it sort of like a local config file for the tool for variables that are unique in value but common to all tools.